web/modules/custom/joinup_invite/joinup_invite.behat.inc

```diff
@@ -49,7 +48,17 @@ class JoinupInviteSubContext extends DrupalSubContextBase implements DrupalSubCo
 
     $invitation_info = $this->getInvitationInfo($type);
     $invitation = Invitation::loadByEntityAndUser($entity, $user, $invitation_info['invitation_bundle']);
-    $arguments = $this->getInvitationMessageHelper()->getMessage($invitation, $invitation_info['message_template'])->getArguments();
+
+    $messages = \Drupal::entityTypeManager()->getStorage('message')->loadByProperties([
+      'template' => $invitation_info['message_template'],
+      'field_invitation' => $invitation->id(),
+    ]);
```

Comment:
We should not put this logic inside a Behat test, we should make a service for this.